OM-Demo/SceneDelegate.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SceneDelegate.swift
+//  OM-Demo
+//
+//  Created by Garo Hussenjian on 1/10/20.
+//
+
+import UIKit
+
+@available(iOS 13.0, *)
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+        guard let _ = (scene as? UIWindowScene) else { return }
```

Comment:
Why do you need this line? Doesn't seem to be doing anything.

Reply:
Certainly that line can be removed. My "loose" intent was not to touch the default app/scene delegate comments and the generated code (with the exception of configuring the scene configuration in the app delegate code). 

This no-op line is theoretically useful for debugging, if only to remind devs that a cast is required to inspect the scene for a window. I'm happy to remove it, but tempted to leave it behind for some pedagogical benefit? Maybe we comment it out rather than delete?

Reply:
Ah, I see, that was not clear to me. Either way works: all the boilerplate left untouched, or all the unneeded boilerplate removed.